1 - identify organisations in the corpus/gather_texts.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #76.                    with open('../data-files/unprocessed/'+section+"/"+str(index)+'.txt', 'w', encoding='utf-8') as f:
</pre><p> </p><p>I needed to add <code>encoding='utf-8'</code> here to make this work on Windows. [Required]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Resolved